btcpay/client.py

```diff
@@ -144,6 +144,23 @@ def create_client(cls, code, host):
         token = client.pair_client(code)
         return BTCPayClient(host=host, pem=pem, tokens=token)
 
+    def create_tor_client(cls, code, host, proxy='socks5://127.0.0.1:9050'):
```

Comment:
I think we need to insert `@classmethod` before this method, no?

Reply:
Good point, I thought I had done so. My mistake.